paramiko_jump/client.py

```diff
@@ -23,7 +23,7 @@
 )
 
 from paramiko.client import SSH_PORT, SSHClient
-
+from paramiko import AutoAddPolicy
```

Comment:
this shuold be imported before our own stuff

Reply:
That makes sense!